core/src/main/java/edu/wpi/grip/core/events/EventLogger.java

```diff
@@ -0,0 +1,23 @@
+package edu.wpi.grip.core.events;
+
+import com.google.common.eventbus.Subscribe;
+import com.google.inject.Singleton;
+
+import java.util.logging.Logger;
+
+/**
+ * Class for logging events as they're posted to the event bus.
+ */
+@Singleton
+public class EventLogger {
+
+  private static final Logger logger = Logger.getLogger(EventLogger.class.getName());
+
+  @Subscribe
+  public void eventPosted(LoggableEvent event) {
```

Comment:
Do you want to make it so that this event can be handled by multiple threads simultaneously?
If so you need to add that other annotation (not sure what its called)